portal/static/js/gil.js

```diff
@@ -479,52 +495,65 @@ module.exports = OrgTool = (function() {
+        var self = this;
 
         $.ajax ({
             type: "GET",
-            url: '/api/user/'+userId+"/consent",
+            url: "/api/user/"+userId+"/consent",
             async: false,
             cache: false
         }).done(function(data) {
             if (data.consent_agreements) {
-                var d = data["consent_agreements"];
-                if (d.length > 0) {
-                    d = d.sort(function(a,b){
-                        return new Date(b.signed) - new Date(a.signed); //latest comes first
-                    });
-                    item = d[0];
-                    expired = OT.getDateDiff(item.expires);
-                    if (item.deleted) found = true;
-                    if (expired > 0) found = true;
-                    if (item.staff_editable && item.include_in_reports && !item.send_reminders) suspended = true;
-                    if (!found) {
-                        if (orgId == item.organization_id) {
-                            //console.log("consented orgid: " + orgId)
-                            switch(filterStatus) {
-                                case "suspended":
-                                    if (suspended) found = true;
-                                    break;
-                                case "purged":
-                                    found = true;
-                                    break;
-                                case "consented":
-                                    if (!suspended) {
-                                        if (item.staff_editable && item.send_reminders && item.include_in_reports) found = true;
-                                    };
-                                    break;
-                                default:
-                                    found = true; //default is to return both suspended and consented entries
-                            };
-                            if (found) consentedOrgIds.push(orgId);
-
-                        };
-                    };
-                }
-            };
+              var d = data["consent_agreements"];
+              if (d.length > 0) {
+                  d = d.sort(function(a,b){
+                      return new Date(b.signed) - new Date(a.signed); //latest comes first
+                  });
+                  var item = d[0];
+                  expired = self.getDateDiff(item.expires);
+                  if (item.deleted) {
+                    found = true;
+                  }
+                  if (expired > 0) {
+                    found = true;
+                  }
+                  if (item.staff_editable && item.include_in_reports && !item.send_reminders) {
+                    suspended = true;
+                  }
+                  if (!found) {
+                    if (String(orgId) === String(item.organization_id)) {
+                      //console.log("consented orgid: " + orgId)
```

Comment:
Could you remove this, or have it only log if the log level is high enough (ie debugging on or development system)?